src/test/java/gov/di_ipv_fraud/step_definitions/FraudAPIStepDefs.java

```diff
@@ -38,8 +37,8 @@ public void user_has_the_user_identity_in_the_form_of_a_signed_jwt_string()
 
         String jsonString =
                 getClaimsForUser(
-                        coreStubUrl, "fraud-cri-build", LindaDuffExperianRowNumber);
-        SESSION_REQUEST_BODY = createRequest(coreStubUrl, "fraud-cri-build", jsonString);
+                        coreStubUrl, "fraud-cri-dev", LindaDuffExperianRowNumber);
```

Comment:
I think we need to pass this in as a param so that it passes on environments as well as locally

Reply:
This is now being passed as a parameter.